app/src/components/main/navigationTabs/navigationTabs.stories.jsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+import { NavigationTabs } from './navigationTabs';
+import README from './README.md';
+
+const activeTab = 'general';
+
+const config = {
+  general: {
+    name: 'General',
+    link: '/general',
+    component: <div>General</div>,
+  },
+  notifications: {
+    name: 'Notification',
+    link: '/notification',
+    component: <div>Notification</div>,
+  },
+  bts: {
+    name: 'Bts',
+    link: '/bts',
+    component: <div>Bts</div>,
+  },
+  defect: {
+    name: 'Defects',
+    link: '/defect',
+    component: <div>Defects</div>,
+  },
+  autoAnalysis: {
+    name: 'Auto Analysis',
+    link: '/autoAnalysis',
+    component: <div>Auto analysis</div>,
+  },
+  demoData: {
+    name: 'Demo data',
+    link: '/demoData',
+    component: <div>Demo data</div>,
+  },
+};
+
+storiesOf('Components/Main/NavigationTabs', module)
+  .addDecorator(
+    host({
+      title: 'Navigation tabs component',
+      align: 'center middle',
+      backdrop: 'rgba(70, 69, 71, 0.2)',
+      background: '#fff',
+      height: 'auto',
+      width: 500,
+    }),
+  )
+  .addDecorator(withReadme(README))
+  .add('default state', () => <NavigationTabs />)
+  .add('with config', () => <NavigationTabs config={config} />)
+  .add('with config & activeTab="general"', () => (
```

Comment:
I think it's a good idea to mock redux store with needed location to see the active tab, because now it's not highlighted

Reply:
Done